app/controllers/manage/finishers_controller.rb

```diff
@@ -70,6 +70,12 @@ def update
     end
   end
 
+  def search
+    # Fetch finishers matching the search term (case insensitive)
+    finishers = Finisher.where('chosen_name ILIKE ?', "%#{params[:term]}%").limit(20)
```

Comment:
Check out finishers.where.limit.select.  This is a better way to avoid creating the Ruby object and get the string attributes directly.

Reply:
Thanks for the tip! I've implemented this now.